<ui version="4.0" >
 <class>FormPointPicker</class>
 <widget class="QMainWindow" name="FormPointPicker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Point Picker</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>800</width>
     <height>560</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QtWorldViewWidget" name="wdgView" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QTableView" name="tblPointList" >
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlacePoint" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Place Point</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="1" >
           <widget class="QPushButton" name="btnView2" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="btnView3" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="btnView4" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="btnView5" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QPushButton" name="btnView6" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QPushButton" name="btnView1" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtWorldViewWidget</class>
   <extends>QSplitter</extends>
   <header>QtForm/include/qtworldviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
